src/modules/75-cd/components/PipelineSteps/CloudFormation/ConnectorStep/CFFileStorePath.tsx

```diff
@@ -0,0 +1,229 @@
+          store: Yup.object().shape({
+            spec: Yup.object().shape({
+              paths: pathSchema
+            })
+          })
+        })
+      })
+    })
+  }
+  return (
+    <Layout.Vertical spacing="xxlarge" padding="small" className={css.tfVarStore}>
+      <Text font="large" color={Color.GREY_800}>
+        {title}
+      </Text>
+      <Formik
+        formName="cfFileStore"
+        initialValues={FormatFilePaths(initialValues, isParam, index)}
+        enableReinitialize
+        validationSchema={isParam ? paramSchema : templateSchema}
+        onSubmit={data => {
+          let connector = prevStepData?.spec?.configuration?.templateFile?.spec?.store?.spec?.connectorRef
+          if (isParam) {
+            connector = prevStepData?.spec?.configuration?.parameters[index]?.store?.spec?.connectorRef
+          }
+          onSubmit(data, connector)
+        }}
+      >
+        {({ values }) => {
+          let name = 'spec.configuration.templateFile.spec.store.spec.paths[0].path'
+          let filePaths = values?.spec?.configuration?.templateFile?.spec?.store?.spec?.paths
+          let connector = prevStepData?.spec?.configuration?.templateFile?.spec?.store?.spec?.connectorRef
+          if (isParam) {
+            name = `spec.configuration.parameters.store.spec.paths`
+            filePaths = values?.spec?.configuration?.parameters?.store?.spec?.paths
+            connector = prevStepData?.spec?.configuration?.parameters[index]?.store?.spec?.connectorRef
+          }
+
+          return (
+            <Form>
+              <div className={css.tfRemoteForm}>
+                {connector?.connector?.type !== 'Aws' && (
+                  <ParameterRepoDetails
+                    isParam={isParam}
+                    allowableTypes={allowableTypes}
+                    index={index}
+                    values={{ ...values, connector }}
+                  />
+                )}
+                <div className={cx(stepCss.md)}>
+                  <MultiTypeFieldSelector
+                    name={name}
+                    style={{ width: 370 }}
+                    allowedTypes={allowableTypes.filter(item => item !== MultiTypeInputType.EXPRESSION)}
+                    label={<Text flex={{ inline: true }}>{title}</Text>}
+                  >
+                    {!isParam ? (
+                      <FormInput.MultiTextInput
+                        name={name}
+                        label=""
+                        multiTextInputProps={{
+                          expressions,
+                          allowableTypes: allowableTypes.filter(item => item !== MultiTypeInputType.RUNTIME)
+                        }}
+                      />
+                    ) : (
+                      <FieldArray
+                        name={name}
+                        render={arrayHelpers => (
+                          <>
+                            {map(filePaths, (path: any, i: number) => (
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9232213)
```suggestion
                            {map(filePaths, (path: unknown, i: number) => (
```